tensorflow/stream_executor/rocm/rocm_blas.cc

```diff
@@ -460,17 +460,17 @@ bool ROCMBlas::DoBlasAsum(Stream *stream, uint64 elem_count,
 bool ROCMBlas::DoBlasAsum(Stream *stream, uint64 elem_count,
                           const DeviceMemory<std::complex<float>> &x, int incx,
                           DeviceMemory<float> *result) {
-  LOG(ERROR) << "rocBLAS does not currently support the ASUM operation "
-             << "for the \"complex<float>\" datatype";
-  return false;
+  return DoBlasInternal(wrap::rocblas_scasum, stream,
+                        false /* = pointer_mode_host */, elem_count,
```

Comment:
As a general rule, TF style for named argument comments is:
`/*pointer_mode_host=*/false`

Same everywhere.

Reply:
pushed out a commit to address this